package.json

```diff
@@ -52,10 +53,13 @@
   },
   "dependencies": {
     "adal-node": "^0.2.1",
+    "axios": "^0.21.0",
+    "dotenv": "^8.2.0",
     "graphql": "^15.4.0",
     "graphql-request": "^3.3.0",
     "graphql-tag": "^2.11.0",
     "jsonpath": "^1.0.2",
+    "querystring": "^0.2.0",
```

Comment:
Not used?

Reply:
Removed in 7c07bc9316eaa1106627849e8b2d026c6f47beaf.